lib/challange3.rb

```diff
@@ -0,0 +1,26 @@
+def deposit(money)
+  puts "How much would you like to deposit?"
+  answer = gets.to_i
+  money += answer
+end
+
+def withdraw(money)
+  puts "How much would you like to withdraw?"
+  answer = gets.to_i
+  money -= answer
+end
+
+def check_balance(money)
+  puts "Your current balance is #{money}"
+end
+
+money = 1000
+puts "What would you like to do? (deposit, wthdraw or check_balance)"
+answer = gets.chomp
+
+if answer == "deposit"
+  deposit(money)
+elsif answer == "withdraw"
+  withdraw(money)
+elsif answer == "check_balance"
+end
```

Comment:
Awesome job making each function a .. well.. function :D if you add a do while loop to the call section you could have the application run until the user chooses to exit.